src/components/AuthContainer.js

```diff
@@ -38,6 +39,25 @@ export default class AuthContainer extends Component {
       }, () => {
         console.warn('Error looking up user status')
       })
+
+      const [billingProjects, clusters] = await Promise.all([Rawls.listBillingProjects(), Leo.clustersList()])
```

Comment:
Is there a less convoluted way to do this without one blocking the other?

Reply:
[Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) is the standard mechanism for aggregating multiple promises. What seems convoluted about it?

Reply:
I think mainly the array destructuring from an array construction feels like a double negative to me.